server/filestore.go

```diff
@@ -1709,7 +1779,7 @@ func (fs *fileStore) SubjectsState(subject string) map[string]SimpleState {
 	}
 
 	start, stop := fs.blks[0], fs.lmb
-	// We can short circuit if not a wildcard using psim for start and stop.
+	// We can short circuit if not a wildcard using psim1 for start and stop.
```

Comment:
(Here `psim1` referring to `psim` I think)

Reply:
Fixed thx.